### PR TITLE
Dev hannah

### PR DESCRIPTION
**Pull request by hannahgardner96**
Dashboard - the time summaries section/bar is not properly centered over the time log. See below.

### CODE REVIEW

File: src/components/Timelog/EffortBar.jsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ const EffortBar = ({activeTab, projectsSelected}) => {
     const totalTime = tangibleTime + intangibleTime;
        
     return (
-        <div className="m-2 row text-white text-center">
+        <div style = {{width: "96%", alignSelf: "center"}} className="m-2 row text-white text-center">
```
Comment (calvertwong) [inline src/components/Timelog/EffortBar.jsx:24]: Does it look good on a tablet screen?

Comment (hannahgardner96) [reply src/components/Timelog/EffortBar.jsx:24]: It works above widths of ~300px. There's a margin of 8 applied to it that I'm not sure where is coming from. It doesn't change when I apply an inline margin of 0. Less than a 300px width, it shifts out of the box to the right.

Comment (calvertwong) [reply src/components/Timelog/EffortBar.jsx:24]: If you could edit/attach one or two screenshots that would be great. Margin 8 could be coming from `className="m-2"` and you could do `        <div style = {{width: "96%"}} className="m-2 row text-white text-center align-self-center">`. `align-self-center` is the bootstrap way of applying `alignSelf: "center"` to the style https://getbootstrap.com/docs/4.0/utilities/flex/

Comment (hannahgardner96) [reply src/components/Timelog/EffortBar.jsx:24]: Screen Shot 2021-09-24 at 11 11 45 AM
Screen Shot 2021-09-24 at 11 11 31 AM
I changed my local version to your suggestion and am still seeing the same behavior. These are the widths at which it starts to become apparent. Searching the files for "m-2" I'm not able to find any styling applied. Do you have any suggestions?

Comment (calvertwong) [reply src/components/Timelog/EffortBar.jsx:24]: <div className="d-flex mb-2 flex-md-row flex-column text-white text-center">
            <div className="bg-primary col-md-4 p-1">
                Tangible Effort: {tangibleTime.toFixed(2)} hrs
            </div>
            <div className="bg-secondary col-md-4 p-1">
                Intangible Effort: {intangibleTime.toFixed(2)} hrs
            </div>
            <div className="bg-success col-md-4 p-1">
                Total Effort: {totalTime.toFixed(2)} hrs
            </div>
        </div>
        
The above code should work, check if this works and meet the requirements. Replacing span with div did the trick and also adding some flex properties at the container for responsiveness.

Comment (hannahgardner96) [reply src/components/Timelog/EffortBar.jsx]: Thank you so much for your help! Integrated Pranav's comment and it works great.

File: src/components/Timelog/EffortBar.jsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ const EffortBar = ({activeTab, projectsSelected}) => {
     const totalTime = tangibleTime + intangibleTime;
        
     return (
-        <div className="m-2 row text-white text-center">
+        <div style = {{width: "96%", alignSelf: "center"}} className="m-auto row text-white text-center align-self-center">
```
Comment (calvertwong) [inline src/components/Timelog/EffortBar.jsx:24]: `<div className="m-auto row text-white text-center">`. This is enough for it to work.

File: src/components/Timelog/EffortBar.jsx (hunk 1 above)
Comment (calvertwong) [reply src/components/Timelog/EffortBar.jsx]: `<div className="m-auto row text-white text-center">`. This is enough for it to work. align self center is not required to be here.

Comment (hannahgardner96) [reply src/components/Timelog/EffortBar.jsx]: Removed it!

**Review by calvertwong — APPROVED** [review]

**Pull request merged**